src/vs/base/browser/ui/sash/sash.ts

```diff
@@ -331,29 +339,29 @@ export class Sash extends Disposable {
 	}
 
 	layout(): void {
-		const size = isIPad ? 20 : 4;
-
-		if (this.orientation === Orientation.VERTICAL) {
-			const verticalProvider = (<IVerticalSashLayoutProvider>this.layoutProvider);
-			this.el.style.left = verticalProvider.getVerticalSashLeft(this) - (size / 2) + 'px';
+		if (this.layoutProvider) {
```

Comment:
You can remove this if. I don't really know why the condition is there, it kind of survived our adoption of strict-null-checks... but it can easily be removed now, this can't ever not be defined.